Exiled.Events/Handlers/Server.cs

```diff
@@ -121,20 +121,7 @@ public static class Server
         /// Called when sending a complaint about a player to the local server administrators.
         /// </summary>
         /// <param name="ev">The <see cref="LocalReportingEventArgs"/> instance.</param>
-        public static void OnLocalReporting(LocalReportingEventArgs ev)
-        {
-#if DEBUG
-            void LogL(string s) =>
-                Log.Debug($"{nameof(Server)}-{nameof(OnLocalReporting)}: {s}", true);
-
-            LogL($"Original reason: {ev.Reason}");
-            ev.Reason = "some random reason for debug";
-            LogL($"Replaced reason: {ev.Reason}");
-            LogL($"{nameof(ev.Target)} is null: {ev.Target == null}");
-            LogL($"{nameof(ev.Issuer)} is null: {ev.Issuer == null}");
-#endif
-            LocalReporting.InvokeSafely(ev);
-        }
```

Comment:
Why remove our debug logs?

Reply:
Because are useless, Rebbok coded that transpiler in a wrong way, adding unnecessary code inside and outside.
I talked to @iopietro about it.
If you want to keep those lines, then let me make a PR where I add debug lines inside each event.

Reply:
Tbh that wouldn't be a bad idea, we should have debug lines inside most of the events rather than just adding them as needed when we run into issues.